cmake/developer_package/packaging/rpm/rpm.cmake

```diff
@@ -102,6 +102,10 @@ macro(ov_define_component_include_rules)
     # scripts
     set(OV_CPACK_COMP_INSTALL_DEPENDENCIES_EXCLUDE_ALL EXCLUDE_FROM_ALL)
     set(OV_CPACK_COMP_SETUPVARS_EXCLUDE_ALL EXCLUDE_FROM_ALL)
+    # pkgconfig
+    set(OV_CPACK_COMP_PKG_CONFIG_EXCLUDE_ALL ${OV_CPACK_COMP_CORE_DEV_EXCLUDE_ALL})
+    # symbolic links
+    set(OV_CPACK_COMP_LINKS_EXCLUDE_ALL ${OV_CPACK_COMP_CORE_DEV_EXCLUDE_ALL})
```

Comment:
1. Each component should be defined via `ov_cpack_add_component`
2. I suppose you need to override these components via `ov_override_component_names` to `core_dev` for compatibility. It should be done for all distribution types except archives

Reply:
done